migtests/tests/pg/assessment-report-test/expectedAssessmentReport.json

```diff
@@ -327,6 +329,19 @@
 				}
 			],
 			"DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#view-with-check-option-is-not-supported"
+		},
+		{
+			"FeatureName": "Unlogged tables",
+			"Objects": [
+				{
+					"ObjectName": "public.tbl_unlogged",
+					"SqlStatement": "CREATE UNLOGGED TABLE public.tbl_unlogged (\n    id integer,\n    val text\n);"
+				},
+				{
+					"ObjectName": "schema2.tbl_unlogged",
+					"SqlStatement": "CREATE UNLOGGED TABLE schema2.tbl_unlogged (\n    id integer,\n    val text\n);"
+				}
+			]
```

Comment:
I see you have a TODO for the docs link, but curious - what would the current state of html report be if the DocsLink field is not populated? 
Is it a broken link? Does the details link not show up at all?

Reply:
In case the DocsLink field is empty it is not populated at all in the HTML files e.g. in assessment report the Details won't show up completely.
BTW, added the docs link as well now.